<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportGenerator</class>
 <widget class="QWidget" name="ReportGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="generateReport">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GenerateReport</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadObjectFileButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Object File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSurfaceFileButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Surface File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadPointImageFile_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Point-Image File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="objectFileLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="surfaceFileLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeFileLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadVolumeFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Volume File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pointImageFileLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>0</y>
     <width>20</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="featureList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureNameLineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Centroid (Lon/Lat)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureCentroidLatLineEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureCentroidLonLineEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Area</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureAreaLineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureVolumeLineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parse Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshListButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyToClipboardButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy to Clipboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Object Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="featureTypeComboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CONTAMINED_AREA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEBRIS_PILE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OBJECT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERSON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STRUCTURE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRAIN</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="structureStateComboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DAMAGED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UNDAMAGED</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="objectTypeComboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BOAT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FIRE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TENT</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>771</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="imageList">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image List</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureBarCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bar code</string>
   </property>
  </widget>
  <widget class="QListWidget" name="boundingPointList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bounding Point List</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
